dawn/renderer/components/peripherals/BooleanSensor.js

```diff
@@ -1,51 +1,42 @@
+/* eslint-disable camelcase */
 import React from 'react';
+import PropTypes from 'prop-types';
 import _ from 'lodash';
 import { PeripheralTypes } from '../../constants/Constants';
 
 /**
- * A component representing a boolean sensor,
- * for example a LimitSwitch.
+ * Boolean Sensor Component
  */
-class BooleanSensor extends React.Component {
-  /**
-   * Formats data for display based on peripheralType.
-   */
-  static formatBoolean(peripheralType, sensorValue) {
-    if (peripheralType === PeripheralTypes.LimitSwitch) {
-      return (sensorValue) ? 'Closed' : 'Open';
-    }
-    return sensorValue;
+const formatBoolean = (peripheralType, sensorValue) => {
+  if (peripheralType === PeripheralTypes.LimitSwitch) {
+    return (sensorValue) ? 'Closed' : 'Open';
   }
+  return sensorValue; // TODO: Verify if int_value or bool_value
+};
```

Comment:
This function seems a bit strange; the `if` condition expects a boolean (or truthy `sensorValue`) and returns a string, while the `else` branch expects an int (I think)?